terraform/lambda/parameter-store.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_ssm_parameter" "credential-issuers-config" {
+  name  = "/${var.environment}/credential-issuers-config"
```

Comment:
The param store has an idea of using paths to namespace stuff, so you could pull all params under `/dev/` for example. We've said in the ADR that we want to namespace params in case we want to deploy multiple envs to the same account. I figured using the paths syntax will give us that as well as being able to use the hierarchy if needed.

Reply:
https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-hierarchies.html